diff_drive_controller/src/diff_drive_controller.cpp

```diff
@@ -372,7 +372,7 @@ namespace diff_drive_controller
     setOdomPubFields(root_nh, controller_nh);
 
     // Set dynamic reconfigure params defaults to the static params provided:
-    DiffDriveControllerConfig config;
+    DiffDriveControllerConfig config(config_default_);
```

Comment:
Note also that there's a long-standing ticket about this on dynamic_reconfigure: https://github.com/ros/dynamic_reconfigure/issues/33